service-authorization/src/main/java/org/lindeck/authorization/controllers/MainController.java

```diff
@@ -0,0 +1,15 @@
+package org.lindeck.authorization.controllers;
+
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/")
+public class MainController {
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    String test() {
+        return "test";
+    }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/45079123/99844731-975c0600-2b84-11eb-96c0-b01d309ef493.png)
where is Test :<

Reply:
Did you try to sign-in?

Reply:
go push the last updates I will test

Reply:
Try to sign-in!